crates/ol/stf/src/tests_snark.rs

```diff
@@ -86,6 +234,21 @@ fn test_snark_account_deposit_and_withdrawal() {
         "Account balance should reflect the deposit"
     );
 
+    // Check inbox state after genesis
+    let snark_state_after_genesis = account_after_deposit.as_snark_account().unwrap();
+    let inbox_idx_after_genesis = snark_state_after_genesis.next_inbox_msg_idx();
+    eprintln!(
+        "DEBUG: Inbox MMR has {} messages (next insert at index {})",
+        inbox_idx_after_genesis, inbox_idx_after_genesis
+    );
+
+    // Check the proof state (next message to PROCESS)
+    let current_processing_idx = snark_state_after_genesis.inner_state_root(); // Wrong field, but let's see
+    eprintln!(
+        "DEBUG: Current inner_state_root: {:?}",
+        current_processing_idx
+    );
```

Comment:
Use the `debug!` macro instead of `eprintln!`.
Also you can add all these inline.

```suggestion
    debug!(
        "Inbox MMR has {inbox_idx_after_genesis} messages (next insert at index {inbox_idx_after_genesis})");

    // Check the proof state (next message to PROCESS)
    let current_processing_idx = snark_state_after_genesis.inner_state_root(); // Wrong field, but let's see
    debug!("Current inner_state_root: {current_processing_idx:?}");
```